Resources/Prototypes/_Impstation/Entities/Clothing/Eyes/hud.yml

```diff
@@ -0,0 +1,12 @@
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons, BaseSyndicateContraband]
+  id: ClothingEyesHudSyndicateSec
+  name: syndicate peacekeeper visor
+  description: The syndicate's OTHER professional heads-up display, designed for better detection of humanoids and their subsequent elimination. This one's flashproof!
```

Comment:
Syndicate is a proper noun and should be capitalized. check this across all altered files please